doc/040_backup.rst

```diff
@@ -315,42 +315,47 @@ Including Files
 ***************
 
 The options ``--files-from``, ``--files-from-verbatim`` and ``--files-from-raw``
-allow you to list files that should be backed up in a file, rather than on the
-command line. This is useful when a lot of files have to be backed up that are
-not in the same folder.
+allow you to give restic a file containing lists of file patterns or paths to
+be backed up. This is useful e.g. when you want to back up files from many
+different locations, or when you use some other software to generate the list
+of files to back up.
 
 The argument passed to ``--files-from`` must be the name of a text file that
-contains one pattern per line. The file must be encoded as UTF-8, or UTF-16
+contains one *pattern* per line. The file must be encoded as UTF-8, or UTF-16
 with a byte-order mark. Leading and trailing whitespace is removed from the
-patterns. Empty lines and lines starting with a ``#`` are ignored.
-The patterns are expanded, when the file is read, by the Go function
-`filepath.Glob <https://golang.org/pkg/path/filepath/#Glob>`__.
-
-The option ``--files-from-verbatim`` has the same behavior as ``--files-from``,
-except that it contains literal filenames. It does expand patterns; filenames
-are listed verbatim.  Lines starting with a ``#`` are not ignored; leading and
-trailing whitespace is not trimmed off. Empty lines are still allowed, so that
-files can be grouped.
-
-``--files-from-raw`` is a third variant that requires filenames to be terminated
-by a zero byte (the NUL character), so that it can even handle filenames that
-contain newlines or are not encoded as UTF-8 (except on Windows, where the
-listed filenames must still be encoded in UTF-8).
```

Comment:
I think we should keep the UTF-8 remark. Or just state that we assume the file to be UTF-8 on Windows. Windows uses UTF-16 in too many places so it's worth describing that UTF-8 is required.